packages/core/src/VNodes/SimpleElementNode.ts

```diff
@@ -1,7 +1,54 @@
 import { VNode, VNodeType } from './VNode';
 
 export class SimpleElementNode extends VNode {
+    htmlTag = '';
     constructor(type: VNodeType) {
         super(type);
+        this.htmlTag =
+            {
+                heading1: 'H1',
+                heading2: 'H2',
+                heading3: 'H3',
+                heading4: 'H4',
+                heading5: 'H5',
+                heading6: 'H6',
+                paragraph: 'P',
+            }[this.type] || 'UNKNOWN-ELEMENT';
```

Comment:
May be use a const mapping between `VNodeType.HEADING1` and `H1`
(because the value of VNodeType.HEADING1 can be change in the future)